.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: on release hook
+on:
+  create:
+    branches:
+      - 'release/*'
```

Comment:
for releases, we don't use `release/*` naming; but just tags,  like https://github.com/tlsnotary/tlsn/blob/main/.github/workflows/ci.yml#L7-L8